README.md

```diff
@@ -1,75 +1,49 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Project Name 
+To do list.
 
-> Description the project.
+# Description of project.
+This To-do list App is built using webpack and served by a webpack dev server
 
+## Project web demo link. 
 
-## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+Built With Html. Css. JS.
 
-## Live Demo (if available)
+Website link: 
 
-[Live Demo Link](https://livedemo.com)
+# Getting starting 
+To get a local copy up and running follow these simple example steps:
 
+## Prerequisites
+ Vscode. 
+ must have git installed.
 
-## Getting Started
+## Setup 
+Clone this repo with:
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add a more comprehensive guide on how to get started with your project.
For Example:

## EXAMPLE
```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/eerapheal/toDolist/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/eerapheal/toDolist.git
```
```
- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```
```
2. Start the development server

   ```sh
   npm start
   ```
```
3. Open [http://localhost:8080](http://localhost:8080) to view it in the browser.

## 🚀 Building and Running for Production

1. Generate a full static production build

   ```sh
   npm run build
   ```

Reply:
Thanks @topeogunleye